apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -44,9 +44,13 @@ var ColorPickerPropertyRow = React.createClass({
       verticalAlign: 'top'
     };
     let colorPicker = this.state.displayColorPicker ? (
-      <ColorPicker type="sketch"
-                   color={this.state.value}
-                   onChangeComplete={this.handleColorChange}/>
+      <div style={{position: 'absolute'}}>
+        <div style={{position: 'fixed', top: '0', right: '0', bottom: '0', left: '0'}}
+             onClick={this.toggleColorPicker}></div>
```

Comment:
Yep, I did try that. And it didn't work. And then I tried to figure out why it didn't work, and AFAICT, the `display` property isn't actually hooked up to anything. I think the documentation might be out of date. I filed an issue with the react-color project: https://github.com/casesandberg/react-color/issues/109.